coala_quickstart/generation/Settings.py

```diff
@@ -52,11 +52,21 @@ def generate_ignore_field(project_dir,
         A comma-separated string containing the globs to ignore.
     """
 
+<<<<<<< HEAD
```

Comment:
well this doesnt look right, does it ;-)

edit the file, amend your commit, repush.